src/components/MovieDetail/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import './styles.css';
+
+function MovieDetail({ movieInfo }) {
+  console.log('SELECTED MOVIE: ', movieInfo);
+  const releaseYear = movieInfo.release_date.split('-')[0];
+
+  useEffect(() => {
+  });
+
+  return (
+    <div className="movieDetail h-[600px] w-full">
+      <div className="bg-cover bg-center bg-top h-[600px] w-full opacity-70" style={{ backgroundImage: `url(https://image.tmdb.org/t/p/original/${movieInfo.backdrop_path})` }} />
```

Comment:
you can set the background image using tailwind using `bg-[url(<path-to-image>)]`